extensions/amp-subscriptions/0.1/local-subscription-platform-renderer.js

```diff
@@ -123,6 +127,15 @@ export class LocalSubscriptionPlatformRenderer {
         if (expr && evaluateExpr(expr,
             /** @type {!JsonObject} */(renderState))) {
           candidate.classList.add('i-amphtml-subs-display');
+          if (candidate.getAttribute('subscriptions-service')
+            && candidate.getAttribute('subscriptions-action')
+            && candidate.hasAttribute('subscriptions-decorate')) {
```

Comment:
Can we make it a default? I think it's better to disable then explicitly enable. E.g. `if (... && candidate.getAttribute('subscriptions-decorate') !== 'false')`

Reply:
done